ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/general/">
+<chapter>
+<title>Maintaining Your Package</title>
+
+<section>
+<title>General responsibilities</title>
+<body>
+
+<note>
+Gentoo's Quality Assurance (QA) team also maintains a
+<uri link="https://projects.gentoo.org/qa/policy-guide/">policy guide</uri>,
+which contains useful advice on ebuild structure and behaviour.
+</note>
+  
+<p>
+You should always use the latest <uri link="::ebuild-writing/eapi/">EAPI</uri>.
+
+The main exception is for critical packages like Portage or
+those typically maintained by base-system which normally wait
+~2-3 years.
+
+Note that for new EAPIs, we require 1 year before using them in
+stable ebuilds.
+
+Another common exception is when an eclass you consume does not yet support it.
+You should file a bug or otherwise contact the eclass maintainer to enquire
+about its status with respect to the new EAPI.
+
+<e>Rationale: Newer EAPIs add improvements which are often user-visible,
+e.g. EAPI 6's eapply_user and EAPI 7's improved cross-compile support.</e>
+</p>
+
+<p>
+Check regularly for new bugs on
+<uri link="https://bugs.gentoo.org/">bugs.gentoo.org</uri>.
+</p>
+
+<p>
+Keep an eye out for CI/tinderbox style bugs too, which are found by testing the
+latest ebuilds (by commit) and random packages, respectively.
+</p>
+
+<p>
+Address CI warnings for your packages. These are generated by pkgcheck.
+You can run <c>pkgcheck scan</c> on your packages to detect most of these,
+but some only become visible when running the scans on the whole tree
+(e.g. broken dependency graph).
+
+Please note that repoman, while advisable to use, detects different
+(mostly a subset, with one or two exceptions) issues from what pkgcheck can.
+Given pkgcheck powers Gentoo's CI, it is wise to make use of pkgcheck.
+
+You can view the current full CI report
+<uri link="https://qa-reports.gentoo.org/output/gentoo-ci/output.html">
+here</uri>.
+
+The CI bot should automatically email you if you are responsible for new
+warnings.
+
+Ask a fellow developer if you're unsure of the meaning or how to fix a warning.
+If in doubt about who to ask, head to the #gentoo-dev, #gentoo-qa, or
+#gentoo-dev-help (for non-developers too) IRC channels.
+</p>
+
+<p>
+Most Gentoo developers tend to agree that pkgcheck in its own serves the QA needs
```

Comment:
Citation needed :P I doubt anyway would refute the technical superiority of pkgcheck. Perhaps we should work around this controversy by simply stating that fact and telling that irrespective of the tool being used, it's the developer's responsibility to ensure quality control. This way we can keep this paragraph short and to the point.